app/models/responses/ClientClaimResponse.scala

```diff
@@ -16,15 +16,23 @@
 
 package models.responses
 
-import play.api.libs.json.{Json, OFormat}
+import models.responses.ClientClaimResponse.invalidCaseRegex
+import play.api.libs.json.{Format, Json, OFormat}
 import uk.gov.hmrc.nationaldutyrepaymentcenter.models.responses.ApiError
 
-final case class ClientClaimSuccessResponse(
-  correlationId: String,
-  caseId: Option[String],
-  error: Option[ApiError] = None
-)
+final case class ClientClaimResponse(correlationId: String, caseId: Option[String], error: Option[ApiError] = None) {
+  val isSuccess    = error.isEmpty
+  val isNotFound   = error.flatMap(_.errorMessage).exists(_.matches(invalidCaseRegex))
+  val isKnownError = isNotFound // || isCaseClosed
```

Comment:
Is the comment here deliberate?

Reply:
Yes - just so you didn't ask why I created a new variable that mirrors an existing one!  ;-)